### PR TITLE
Jc pickup weapon

### PR DESCRIPTION
**Pull request by jcode89**
The official pull request for issue #53

### CODE REVIEW

File: modules/command_line.py [hunk 1]
```diff
@@ -63,8 +63,10 @@ def map_key(self):
         %s: Key\n
         %s: Exit\n
         %s: Player\n
-        %s: Gold
-        """ % (">", "~", "<", "@", "$"))
+        %s: Gold\n
+        %s: Cockroach\n
```
Comment (jamalhansen) [inline modules/command_line.py]: Nice, you got the roach too

Comment (jcode89) [inline modules/command_line.py]: I couldn't forget about the roach :smile:

**Comment by jamalhansen** [comment]
Looks good to me

**Comment by jcode89** [comment]
Uh-oh does that mean it's dancing time :smile:

**Pull request merged**